### ISSUE TITLE
[Shattered Sun Offensive] One-time quests are dailies

### ISSUE THREAD

**Issue by Ottawia** [chromiecraft/chromiecraft#7481]
### What client do you play on?

enGB

### Faction

Alliance

### Content Phase:

Generic

### Current Behaviour

Quests that should not be dailies are dailies here screenshoots i have done next day after did all quests day before: 
![image](https://github.com/user-attachments/assets/be78c3cc-4f3d-4b8b-9cd8-51918c99ce54)
![image](https://github.com/user-attachments/assets/05a4c93d-f0ab-4616-b850-328114ee535d)
![image](https://github.com/user-attachments/assets/b0b7f0df-97d0-4584-bc49-79c30a8cb76e)
![image](https://github.com/user-attachments/assets/efc2f2fc-abcb-40e2-a573-36da1232f46c)
![image](https://github.com/user-attachments/assets/2967f338-9582-4e23-b390-8971c5755ae6)

### Expected Blizzlike Behaviour

One of each duble quests suppose to be one-time quest and be replaced by daily (other quest) after returning.

### Source

Sun's reach armory: https://warcraft.wiki.gg/wiki/The_Battle_for_the_Sun%27s_Reach_Armory
Distraction at the Dead Scar: https://wowpedia.fandom.com/wiki/Distraction_at_the_Dead_Scar
Making Ready: https://wowpedia.fandom.com/wiki/Making_Ready
Discovering Your Roots: https://wowpedia.fandom.com/wiki/Discovering_Your_Roots
Intercept the Reinforcements: https://wowpedia.fandom.com/wiki/Intercept_the_Reinforcements

### Steps to reproduce the problem

1. Get to isle of Quel'danas on lvl 70 character with aldor/scryer rep
2. Do quests that suppose to not be dailies
3. Return next day
4. Check if you can do one-time-quests again

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/b45c41cf6ebe9a715bed8dfe4257acc66cca8efd

### Operating system

Ubuntu 22.04

### Modules

- [mod-1v1-arena](https://github.com/azerothcore/mod-1v1-arena)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-anticheat](https://github.com/azerothcore/mod-anticheat)
- [mod-arena-3v3-solo-queue](https://github.com/azerothcore/mod-arena-3v3-solo-queue)
- [mod-arena-replay](https://github.com/azerothcore/mod-arena-replay)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-breaking-news-override](https://github.com/azerothcore/mod-breaking-news-override)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-costumes](https://github.com/azerothcore/mod-costumes)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-dmf-switch](https://github.com/azerothcore/mod-dmf-switch)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-npc-spectator](https://github.com/gozzim/mod-npc-spectator)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-eventScripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-ExchangeNpc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-RecruitAFriend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-SendAndBind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-TempAnnouncements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-Zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by heyitsbench** [comment]
This is not true. The issue here is that the dailies have both of their phase versions available at a given time, i.e. as the Isle progresses, dailies become available/swap IDs. This will be fixed my Isle progression PR hopefully shortly.

**Comment by Rorschach91** [comment]
We'll wait you fix @heyitsbench :)

**Comment by TheSCREWEDSoftware** [comment]
@heyitsbench was this fixed with the isle progression yet?

**Comment by Rorschach91** [comment]
> [@heyitsbench](https://github.com/heyitsbench) was this fixed with the isle progression yet?

Yes, it is. It can be closed.
